src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -327,14 +325,26 @@ model {
 ```
 
 In this second model, the parameters `x_raw` and `y_raw` are
-sampled as independent standard normals, which is easy for Stan.  These
-are then transformed into samples from the funnel.  In this case, the
-same transform may be used to define Monte Carlo samples directly
-based on independent standard normal samples; Markov chain Monte Carlo
-methods are not necessary. If such a reparameterization were used in
-Stan code, it is useful to provide a comment indicating what the
-distribution for the parameter implies for the distribution of the
-transformed parameter.
+sampled as independent standard normals, which is easy for Stan,
+and then transformed into samples from the funnel.
+When this reparameterization is used in Stan code, a comment indicating what the
```

Comment:
Although they are technically reparameterizations under the hood, I think we should stick to calling them transforms.  The terminology is confusing if you have any suggestions---the problem is that with the affine transform there is no constraining or unconstraining.  The domain and range of the transform function are the same---it's just a bijection over the whole real number line.